cuebot/src/main/resources/opencue.properties

```diff
@@ -11,6 +11,7 @@ datasource.cue-data-source.maxAge=21600000
 
 grpc.cue_port=${CUEBOT_GRPC_CUE_PORT:8443}
 grpc.rqd_server_port=${CUEBOT_GRPC_RQD_SERVER_PORT:8444}
+grpc.use_fqdn=false
```

Comment:
Please add a comment describing what this setting does.

Reply:
Added